roles/setup_gitea/tasks/install.yml

```diff
@@ -0,0 +1,337 @@
+---
+- name: Create namespace for Gitea
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ sg_namespace }}"
+
+- name: Create Service Account
+  community.kubernetes.k8s:
+    definition:
+      kind: ServiceAccount
+      apiVersion: v1
+      metadata:
+        name: anyuid
+        namespace: "{{ sg_namespace }}"
+
+- name: Create Role
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: Role
+      metadata:
+        name: scc-anyuid
+        namespace: "{{ sg_namespace }}"
+      rules:
+        - apiGroups:
+            - security.openshift.io
+          resourceNames:
+            - anyuid
+          resources:
+            - securitycontextconstraints
+          verbs:
+            - use
+
+- name: Create RoleBinding
+  community.kubernetes.k8s:
+    definition:
+      kind: RoleBinding
+      apiVersion: rbac.authorization.k8s.io/v1
+      metadata:
+        name: sa-to-scc-anyuid
+        namespace: "{{ sg_namespace }}"
+      subjects:
+        - kind: ServiceAccount
+          name: anyuid
+      roleRef:
+        kind: Role
+        name: scc-anyuid
+        apiGroup: rbac.authorization.k8s.io
+
+- name: Generate JWT secret
+  ansible.builtin.command: /usr/bin/podman run -ti {{ sg_gitea_image }} /usr/local/bin/gitea generate secret JWT_SECRET
+  changed_when: false
+  register: _sg_jwt_secret
+
+- name: Generate the LFS JWT secret
+  ansible.builtin.command: /usr/bin/podman run -ti {{ sg_gitea_image }} /usr/local/bin/gitea generate secret LFS_JWT_SECRET
```

Comment:
Maybe add  `--rm` to all the podman executions

Reply:
Why not use podman module?

Reply:
Thanks for the suggestion, @betoredhat.
@reihl, the thing is we need to register the output of the command, and when I did my initial tests with the podman module I didn't seem to get the outputs right. However, I just repeated the tests and it worked fine so I'll move to use the podman_container module. Thanks for your input.